### PR TITLE
Remove EOL notice

### PR DESCRIPTION
**Pull request by Newbytee**
We have new maintainers.

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build

File: org.seul.pingus.json [hunk 1]
```diff
@@ -14,7 +14,17 @@
         "--socket=pulseaudio"
     ],
     "modules": [
-        "shared-modules/sigc++/sigc++-2.0.json",
+        {
+        	"name": "libsigc++-2.0",
```
Comment (AsciiWolf) [inline org.seul.pingus.json:18]: We could probably update the other modules (including shared-modules) as well as a part of this PR.

**Comment by Newbytee** [comment]
> bot, build

It's not going to work because Grumbel removed a bunch of repositories that building this depends on. We can fix it, but it needs work.

**Comment by AsciiWolf** [comment]
> It's not going to work because Grumbel removed a bunch of repositories that building this depends on. We can fix it, but it needs work.

Oops! I did not notice.

**Comment by Newbytee** [comment]
Oh, come on bot. Anyway, I suggest you look at https://github.com/flathub/org.seul.pingus/pull/6

**Comment by AsciiWolf** [comment]
It looks like that Pingus is now [on GitHub](https://github.com/Pingus/pingus.git) and the AppData file [got removed](https://github.com/Pingus/pingus/commit/9b05d35694956714b9483ab3f974696d3810b2eb), probably because the maintainer erroneously thinks that it is relevant only to Flatpaks. I pointed out to them that they are not correct, but they ignored my ticket.

**edit:** I have made a [new ticket on GitHub](https://github.com/Pingus/pingus/issues/198) about the AppData file removal.

**Comment by AsciiWolf** [comment]
@Newbytee Change the Pingus GitLab repo url to a GitHub one: https://github.com/Pingus/pingus.git

Keep the same Git commit (so we don't have to deal with the removed AppData file for now), just change the repo.

**Comment by AsciiWolf** [comment]
Also, I think that maybe we could merge this PR with #6 to make things easier.

/cc @Newbytee @JakobDev

We need to:
- Remove the EOL notice.
- Change the Pingus repo url to GitHub one.
- Switch from using shared-modules that no longer exist (sadly all of them) to regular modules.
- Update all modules.
- Update the runtime used by this Flatpak to 23.08.

In the future, we will also have to ship the AppStream metadata file with the Flatpak since upstream [removed](https://github.com/Pingus/pingus/commit/9b05d35694956714b9483ab3f974696d3810b2eb) their AppData file. For now, we can stick with the original commit (25d7f3ffbb6ed31de5da7b8da8f9c5fdef75b4f0) instead of bumping it or we can use the commit from before the AppData file removal (2131a2b18325a15eafc0ddd811447a1741e07ac4).

**Comment by AsciiWolf** [comment]
Replaced by #11.

**Pull request closed without merging**